caluma/caluma_workflow/filters.py

```diff
@@ -109,13 +109,18 @@ class CaseOrderSet(BaseFilterSet):
         fields=[
             "created_at",
             "modified_at",
-            "allow_all_forms",
-            "description",
-            "is_archived",
-            "is_published",
-            "name",
             "status",
-            "slug",
+            "workflow__slug",
+            "workflow__name",
+            "workflow__description",
+            "workflow__is_archived",
+            "workflow__is_published",
+            "workflow__allow_all_forms",
+            "document__form__slug",
+            "document__form__name",
+            "document__form__description",
+            "document__form__is_archived",
+            "document__form__is_published",
```

Comment:
I'm not sure those are all useful. It's only about ordering after all. For example, are you sure we will (soon-ish) need to sort cases by the fact whether it's document's form is actually published? Or the form's description? I don't think so. 

Therefore please only add new order-by fields that are either immediately used or easily conceivable of a use case. Most of those are not, IMHO

Reply:
only kept slug and name

Reply:
Still not sure we really need those